src/KafkaFlow.Retry/Durable/Polling/Jobs/RetryDurableActiveQueuesCountJob.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Threading.Tasks;
+using KafkaFlow.Retry.Durable.Definitions.Polling;
+using KafkaFlow.Retry.Durable.Polling.Extensions;
+using KafkaFlow.Retry.Durable.Repository;
+using KafkaFlow.Retry.Durable.Repository.Actions.Delete;
+using KafkaFlow.Retry.Durable.Repository.Actions.Read;
+using KafkaFlow.Retry.Durable.Repository.Model;
+using Quartz;
+
+namespace KafkaFlow.Retry.Durable.Polling.Jobs;
+
+[DisallowConcurrentExecution]
+internal class RetryDurableActiveQueuesCountJob : IJob
+{
+    public async Task Execute(IJobExecutionContext context)
+    {
+        var jobDataMap = context.JobDetail.JobDataMap;
+
+        var retryDurableActiveQueuesCountPollingDefinition =
```

Comment:
Maybe I am being a bit nit-picking, but should we consider cloning this polling definition before using? You are assigning a value of active queues number bellow, and that information will be shared across other logic and threads. Maybe it will never be an issue in this case because the polling definition is only used here, but we should consider that this implementation may serve as role model for other community contributions and it may be a problem on those contributions' case.

This is only a problem because the polling definition is a reference type, if we considered using a struct, it wouldn't be a problem.

Reply:
The `RetryDurableActiveQueuesCountPollingDefinition` is not mutable. We built it through construction, and there are no sets.

Reply:
Understood, the ActiveQueues is not a method, it is an action exposed via a property, it bad interpretation on my part 👍